src/BlazorDB/wwwroot/blazorDB.js

```diff
@@ -149,6 +161,74 @@ window.blazorDB = {
             });
         });
     },
+    getFirstItem: function (dotnetReference, transaction, item) {
+        return new Promise((resolve, reject) => {
+            window.blazorDB.getTable(item.dbName, item.storeName).then(table => {
+                var item = table.toCollection().first();
+                dotnetReference.invokeMethodAsync('BlazorDBCallback', transaction, false, 'getFirstItem succeeded');
+                resolve(item);
+            }).catch(e => {
+                console.error(e);
+                dotnetReference.invokeMethodAsync('BlazorDBCallback', transaction, true, 'getFirstItem failed');
+                reject(e);
+            });
+        });
+    },
+    getMinIndex: function (dotnetReference, transaction, item) {
+        return new Promise((resolve, reject) => {
+            window.blazorDB.getTable(item.dbName, item.storeName).then(table => {
+                table.toCollection().count().then((result) => {
+                    dotnetReference.invokeMethodAsync('BlazorDBCallback', transaction, false, 'getMinIndex succeeded');
+                    resolve(result == 0 ? "Table empty" : 0);
```

Comment:
What about returning a `-1` instead of `Table Empty`?

The concern is just getting it the same type return for `if/else`

Reply:
Implemented.